README.md

```diff
@@ -694,6 +694,35 @@ _NOTE:_ The field-level setting overrides the global config setting (for the fie
 
 </details>
 
+<details>
+<summary>Discarding nil fields</summary>
```

Comment:
**suggestion**: Slight nitpick, but I'd suggest framing this as "omit" or "exclude" instead of "discard". 

```suggestion
<summary Exclude Fields with nil Values</summary>
```